### PR TITLE
added script for FXV configurations

### PR DESCRIPTION
**Pull request by naren076**

### CODE REVIEW

File: SAP/Script/FXVAirHandlingConfiguration.py [hunk 1]
```diff
@@ -94,16 +94,14 @@
     page.WaitConfirm(1000)
   
   #Fan Motor RPM B
-  if air_handling_values["fan_motor_rpm_b"] != "SAP Defined" :
-   if air_handling_values["fan_motor_rpm_b"] != "Not Visible": 
+  if(air_handling_values["fan_motor_rpm_b"] != "SAP Defined" and  air_handling_values["fan_motor_rpm_b"] != "Not Visible"):
```
Comment (naren076) [inline SAP/Script/FXVAirHandlingConfiguration.py:97]: Remove one space after 'and' there is a two space.

File: SAP/Script/FXVProduct.py [hunk 2]
```diff
@@ -105,7 +104,7 @@ def sap_test():
       
     #Price  
     price = frame.FindElement("#configurationComponent---configurationView--headerContainer-2--headerFieldPrice").text
-    record_price(price, record_no_values(sap_field_values), general_requirement_values(sap_field_values))
+    record_price(price,sales_order_values(sap_field_values),general_requirement_values(sap_field_values))
```
Comment (naren076) [inline SAP/Script/FXVProduct.py:107]: user direct Count value

File: SAP/Script/FXVProduct.py [hunk 3]
```diff
@@ -120,29 +119,17 @@ def sap_test():
...
-    textbox.FindElement("//div[.='Execute']").Click()
     
-    frame = panel.frameApplicationBillofmaterialMu
-    form = frame.formWebguiform0
-    if(NameMapping.Sys.browser.pageFlp.frameApplicationBillofmaterialMu.WaitNamedChild("panelSpreadsheetCtrlShiftF7", 10000).Exists):
-     form.FindElement("//div[@title='Spreadsheet... (Ctrl+Shift+F7)']").Click()
-    if(NameMapping.Sys.browser.pageFlp.frameApplicationBillofmaterialMu.WaitNamedChild("panelUpdowndialogchoose", 10000).Exists):
-      frame.FindElement("//div[.='OK']").Click()
+    #Multi-Level Sales Order BOM
+    MultiLevelBOM.multi_level_bom(panel,textbox,page,standard_order_values(sap_field_values))
```
Comment (naren076) [inline SAP/Script/FXVProduct.py:129]: Use direct BOM material value.

File: SAP/Script/FXVProduct.py [hunk 4]
```diff
@@ -341,15 +320,15 @@ def shipping_values(sap_field_values):
   
 
   
-def record_price(price, record_no_values, general_requirement_values):
+def record_price(price,sales_order_values,general_requirement_values):
```
Comment (naren076) [inline SAP/Script/FXVProduct.py:323]: change the parameter name to rec_no

File: SAP/Script/Unit2.py [hunk 5]
```diff
@@ -293,7 +283,7 @@ def shipping_values(sap_field_values):
 def record_price(price, rec_no, general_requirement_values):
   
   # Get the sheet of the Excel file
-  excelFile = Excel.Open("C:\\Users\\narayanan.g\\Documents\\test.xlsx")
+  excelFile = Excel.Open("C:\\Users\\narayanan.g\\Documents\\test1.xlsx")
```
Comment (naren076) [inline SAP/Script/Unit2.py:286]: Create excel with the name of BOM_price_list

File: SAP/Script/FXVAirHandlingConfiguration.py (hunk 1 above)
Comment (vaishnavir9501) [reply SAP/Script/FXVAirHandlingConfiguration.py:97]: I have removed the extra space.

File: SAP/Script/FXVProduct.py (hunk 2 above)
Comment (vaishnavir9501) [reply SAP/Script/FXVProduct.py:107]: removed the pervious parameter and pass the direct count's Json value

File: SAP/Script/FXVProduct.py (hunk 3 above)
Comment (vaishnavir9501) [reply SAP/Script/FXVProduct.py:129]: removed the pervious parameter and pass the direct  BOM material's Json value

File: SAP/Script/FXVProduct.py [hunk 6]
```diff
@@ -164,17 +151,11 @@ def ProcessData(browser, page):
...
-def record_no_values(sap_field_values):
-  record_no = {}  
-  record_no['record_no'] = sap_field_values["Count"].strip()
-  return record_no
-  
+   
 # Sales Order Values
 def sales_order_values(sap_field_values):
   sales_order_field_values = {}
+  sales_order_field_values["record_no"] = sap_field_values["Count"].strip()
```
[the remark this answers is not in the document]
Comment (vaishnavir9501) [reply SAP/Script/FXVProduct.py:158]: i have removed this line.

File: SAP/Script/FXVProduct.py (hunk 4 above)
Comment (vaishnavir9501) [reply SAP/Script/FXVProduct.py:323]: changed the parameter as rec_no

File: SAP/Script/FXVProduct.py [hunk 7]
```diff
@@ -254,6 +254,11 @@ def air_handling_values(sap_field_values):
   air_handling_configurations["fan_motor_rpm_a"] = sap_field_values["Fan Motor RPM A"].strip()
   air_handling_configurations["fan_motor_type"] = sap_field_values["Fan Motor Type"].strip()
   air_handling_configurations["fan_motor_quantity_main_a"] = sap_field_values["Fan Motor Quantity - Main A:"].strip()
+  air_handling_configurations["fan_motor_rpm__a_pony"] = sap_field_values["Fan Motor RPM A Pony:"].strip()
+  air_handling_configurations["fan_motor_type_pony"] = sap_field_values["Fan Motor Type - Pony:"].strip()
+  air_handling_configurations["fan_motor_quantity_pony_a"] = sap_field_values["Fan Motor Quantity - Pony A:"].strip()
+  air_handling_configurations["fan_motor_options_a_pony:"] = sap_field_values["Fan Motor Options A Pony:"].strip()
```
Comment (naren076) [inline SAP/Script/FXVProduct.py:260]: @vaishnavir9501 remove this `:` line 260

Comment (vaishnavir9501) [reply SAP/Script/FXVProduct.py]: colon has been removed.

**Pull request merged**